### PR TITLE
2.x auri

### PR DESCRIPTION
**Pull request by aureamunoz**

### CODE REVIEW

**Comment by cescoffier** [comment]
There is a problem with Eureka. The latest spring dependencies are compiled for Java 17. We need a was to support JAva 11 too. 
If it's problematic, let's change our strategy to test eureka. I'm not sure how, but starting Eureka in the same JVM as the tests seems to not be easy in the future (even currently).

**Comment by cescoffier** [comment]
@aureamunoz there is still the spring issue. These spring dependencies are java 17 only, and so cannot be used here. We need to rollback to versions that are compatible with java 11.

**Review by cescoffier — APPROVED** [review]

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -49,7 +49,7 @@ jobs:
     strategy:
       matrix:
         java: [
-          { 'version': '11', opts: '' },
+          { 'version': '11', opts: '-pl ''!service-discovery/eureka''' },
```
Comment (cescoffier) [inline .github/workflows/build.yml:52]: you also need to skip the docs too, as it depends on that artifacts.

**Review by cescoffier — CHANGES_REQUESTED** [review]

**Comment by cescoffier** [comment]
Can you also reorganize the commits?

File: .github/workflows/build.yml (hunk 1 above)
Comment (aureamunoz) [reply .github/workflows/build.yml:52]: yes, I've just seen that. But I don't understand why it fails on windows and not in ubuntu ¿?¿?

Comment (cescoffier) [reply .github/workflows/build.yml:52]: Pretty sure it's a question of cache.

**Review by cescoffier — APPROVED** [review]
Let's see what the CI says.

**Comment by aureamunoz** [comment]
I can't believe it :sweat_smile:
Ups, I thought having chosen the 2.x branch as base, how do we proceed?

**Comment by cescoffier** [comment]
Let me merge it manually tomorrow. I need to create a 1.x maintenance branch first.

**Comment by cescoffier** [comment]
Manually merged.

**Pull request closed without merging**